src/main/java/seedu/duke/util/StringConstants.java

```diff
@@ -109,6 +116,12 @@
             + "Available commands: exit, add, del, list, mark, save, help";
     public static final String HELP_EXCEPTION = "Sorry, but no help exists for that command.";
 
+    public static final String OPTION_HELP = "Set customized configuration\n"
+            + "Format to set an option: option CONFIGURATION_GROUP=NEW_VALUE\n"
+            + "Check configuration seting: option\n"
```

Comment:
Maybe `"Format to check configuration settings: option\n"` to keep the help cohesive?

Reply:
Modified